CONTRIBUTING.md

```diff
@@ -82,6 +82,29 @@ The checksum can be updated automatically with the `xgensum` helper from the [xt
 
     $ xgensum -i <pkgname>
 
+### Adopting a template
+
+If a template is orphaned (maintained by `orphan@voidlinux.org`) or the current `maintainer` has not contributed to
```

Comment:
```suggestion
If a template is orphaned (maintained by `orphan@voidlinux.org`) or the current maintainer has not contributed to
```

Reply:
I code-quoted it to indicate that it's the variable in the template

Reply:
Fine then.